src/Services/Git.php

```diff
@@ -165,6 +159,17 @@ public function getTracking($includeBranch = false): ?string
         return explode('/', $result)[0];
     }
 
+    /**
+     * @param string $command
+     * @param array ...$argsAndOptions
+     *
+     * @return string
+     */
+    public function run(string $command, ...$argsAndOptions): string
```

Comment:
$argsAndOptions can only be an array of strings? Or only an array of arrays? Or also an array of something else?